composer.json

```diff
@@ -122,6 +122,14 @@
             },
             "drupal/votingapi": {
                 "Issue #3341513: Voting Storages does not check access on entity query\n": "https://www.drupal.org/files/issues/2023-07-06/3341513_votingapi_use-access-checks-14.diff"
+            },
+            "drupal/group": {
+                "Missing config schema for condition.plugin.group_type": "https://www.drupal.org/files/issues/2018-12-14/group-group_type_condition_plugin_config_schema-3020554-2.patch",
+                "Ability to use group tokens in node context": "https://www.drupal.org/files/issues/2023-02-17/group-2774827-93.patch",
+                "(Deprecated) This patch should be deleted when OS will integrate group entity query access handlers": "https://raw.githubusercontent.com/goalgorilla/open_social/a6a1c0d2932eba7efcea050aedc910c3acaec223/patches/gnode_grant_access.patch"
+            },
+            "drupal/socialbase": {
+                "Replace non-existing classes according to group module upgrade to 2.x": "https://git.drupalcode.org/project/socialbase/-/merge_requests/199.patch"
```

Comment:
We probably want to merge this with the existing `"drupal/group": {` patches
![Screenshot 2024-03-26 at 12 20 29](https://github.com/goalgorilla/open_social/assets/16667281/a6717387-f35f-459d-9845-50c37be504b8)

Reply:
done